elements/pfe-clipboard/src/pfe-clipboard.js

```diff
@@ -0,0 +1,173 @@
+import PFElement from "../../pfelement/dist/pfelement.js";
+
+class PfeClipboard extends PFElement {
+  static get tag() {
+    return "pfe-clipboard";
+  }
+
+  static get meta() {
+    return {
+      title: "Clipboard",
+      description: "Copy current URL to clipboard."
+    };
+  }
+
+  get templateUrl() {
+    return "pfe-clipboard.html";
+  }
+
+  get styleUrl() {
+    return "pfe-clipboard.scss";
+  }
+
+  static get events() {
+    return {
+      copied: `${this.tag}:copied`
+    };
+  }
+
+  // Declare the type of this component
+  static get PfeType() {
+    return PFElement.PfeTypes.Content;
+  }
+
+  static get properties() {
+    return {
+      iconHidden: {
+        title: "Icon Hidden",
```

Comment:
What would you think of "Hide the icon" for this title instead?  Something worded a little differently for clarity?

Reply:
Agree!